### PR TITLE
Console Math Game finished! 

### PR DESCRIPTION
**Pull request by jcast24**
Used a bit of my own knowledge with C# while doing the challenge portions and would like some feedback, if any. Overall enjoyed the project!

### CODE REVIEW

File: jcast24.MathGame/Engine.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        return result;
+    }
+
+    internal void Division(int numOfGames)
+    {
+        int score = 0;
+        for (int i = 0; i < numOfGames; i++)
+        {
+            var divArray = GetDivisionNumbers();
+            var answer = divArray[0] / divArray[1];
```
Comment (VocalNight) [inline jcast24.MathGame/Engine.cs:145]: Hello! Everything looks good, but i got an exception when selecting the division option due to a division by 0.

image

Could you double check how you are treating the division part of the app?

**Review by VocalNight — CHANGES_REQUESTED** [review]
Hello!

File: jcast24.MathGame/Engine.cs (hunk 1 above)
Comment (jcast24) [reply jcast24.MathGame/Engine.cs:145]: Hi, I believe I may have fixed the issue. Please let me know if any other errors arise. Thank you!

**Review by VocalNight — APPROVED** [review]
Everything is working perfectly now, good job!

Good luck on the next projects.

**Pull request closed without merging**